### PR TITLE
Translate button on page edit actions

### PR DESCRIPTION
**Pull request by mixxorz**
This PR adds the following action buttons:

**Page edit screen**

* Translate this page
* Sync translated pages

**Snippet edit screen**

* Translated this snippet
* Sync translated snippets

Screen Shot 2022-02-15 at 6 28 34 PM

### CODE REVIEW

File: wagtail_localize_rws_languagecloud/tests/helpers.py [hunk 1]
```diff
@@ -44,3 +50,32 @@ def create_test_project_settings(translation_source, translations, **settings_da
     return LanguageCloudProjectSettings.get_or_create_from_source_and_translation_data(
         translation_source, translations, **data
     )
+
+
+def create_editor_user(username="testeditor"):
```
Comment (chris48s) [inline wagtail_localize_rws_languagecloud/tests/helpers.py:55]: Worth noting here that we also have a slightly different function to help with creating test users

https://github.com/zerolab/wagtail-localize-rws-languagecloud/blob/ad7f8c94ead3eef43af6a21544e26353f39b473d/wagtail_localize_rws_languagecloud/tests/test_emails.py#L14-L20

We don't need to address it right now, but as you work on this sprint there might be an opportunity to pull out some nice generic helpers here. Maybe one to have in mind if another ticket comes up that requires creating test users with various permissions.

**Review by chris48s — APPROVED** [review]
Nice work. I've left one comment but it doesn't require changes to merge this PR

File: wagtail_localize_rws_languagecloud/tests/helpers.py (hunk 1 above)
Comment (mixxorz) [reply wagtail_localize_rws_languagecloud/tests/helpers.py:55]: Usually we'd use factory-boy for generating test data, including users. It currently isn't being used on the project. Would you be OK for me to set that up? (On future PRs)

Comment (chris48s) [reply wagtail_localize_rws_languagecloud/tests/helpers.py:55]: Yeah if you've got time to do that in the scope of this work, having some nice factories would be great :+1:

**Pull request merged**